reports/proposal.Rmd

```diff
@@ -30,15 +30,11 @@ Being the largest city in British Columbia, Vancouver is an attractive city to c
 
 # Research Questions and Usage Scenarios
 
+Mary is a new to Vancouver, who is not very familiar with the neighborhoods. She wants to settle down by renting a place in Vancouver, and now she faces the problem of which neighborhoods to choose. One important factor she takes into account is the crime incidents. She wants to compare the safety of neighborhoods in the aspect of least crime incidents. Moreover, since certain types of crimes (*e.g.* homicide) are more serious and life-threatening than others, she is also willing to know the types of crimes that occur most frequently in a particular neighborhood. In case that she only wants to accommodate for a short period, it would be helpful to know the specific crime distribution during that time frame.
 
+When Mary logs on to the "Van Safe app", she can see the summary statistics (*e.g.* the total crime incidents, the top 3 common crime types, etc.) in Vancouver in 2021. She can then select the neighborhood that she wants to know more, and choose a specific range of time using the slider, she will be able to see the distribution of crime types within the time range she chooses. For instance, she sees that during 2021-06 to 2021-08, which will be the actual duration of her residence in Vancouver, "offense again a person" takes place quite often in Strathcona. Therefore, she decides to remove Strathcona from her candidate list of neighborhoods.
```

Comment:
should we keep it consistent as "safe_vancity" here for the name of the app? I like van safe too!

"offense against* a person" 

instead of saying therefore, she decides...maybe we can say "with this information, Mary may decide to remove....

Reply:
Yeah! How about "Safe Vancity App"? Because I assume this will be the title that shows up on webpage, so better to make it more English rather than machine-readable